ui/components/app/transaction-total-banner/transaction-total-banner.stories.js

```diff
@@ -6,7 +6,7 @@ export default {
   id: __filename,
 };
 
-export const basic = () => {
+export const Base = () => {
```

Comment:
Should follow naming conventions outlined in [docs](https://metamask.github.io/metamask-storybook/index.html?path=/story/getting-started-documentation-guidelines--page)

```suggestion
export const DefaultStory = () => {
```

Then rename to default

```js
DefaultStory.storyName = 'Default';
```